tools/cvm-image-rewriter/plugins/98-test/README.md

```diff
@@ -0,0 +1,3 @@
+# Test
+
+This plugin is used to test cloud-init, files, pre-stage and post-stage.
```

Comment:
It's better to clarify this is a testing plugin. Besides, it;s better to add the expected result running the plugin.

Reply:
@ruomengh  Updated. Please review it again. Thanks.